README.md

```diff
@@ -23,6 +23,19 @@ Creating a metapackage is really easy, we will make use of [equivs](http://apt.u
 - Then run: `equivs-build ns-control` to build your metapackage, thats all simple and easy.
 - To add it to meilix follow adding a metapackage to meilix section.
 
+<<<<<<< HEAD
```

Comment:
Rebasing error, seems like there was a merge conflict. Can you please remove these